io.github.sigmasd.IRust.yml

```
@@ -0,0 +1,39 @@
+app-id: io.github.sigmasd.IRust
+
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+
+command: irust
+
```

finish-args is required

---

it seems to be terminal only.

---

```suggestion
finish-args: []
```

This should be enough to make the linter happy.

CLI application aren't really the ideal use case for flatpak.

---

I understand that, my motivation is mostly the exposure the tools get through store, I discovered a lot of cool projects through gnome software.